package.json

```diff
@@ -11,7 +11,11 @@
   },
   "scripts": {
     "build": "ncc build src/index.ts -o dist",
-    "format": "prettier --write src/**/*.ts"
+    "format": "prettier --write src/**/*.ts",
+    "dev:pr": "EVENT_PATH=event.pr.json yarn dev:run",
+    "dev:issue": "EVENT_PATH=event.issue.json yarn dev:run",
```

Comment:
setting the env variable in this way will not work for Windows users. Also `source .env` will not work

Reply:
hmm ok - thanks for pointing. In that case i think I wil go with `dotenv` and then it should work x-platform